graders/java/Dockerfile

```
@@ -4,20 +4,21 @@ LABEL maintainer="jonatan@yorku.ca"
+    && rm -rf /var/lib/apt/lists/*
 
 ENV LANG=en_US.UTF-8
 
 RUN groupadd sbuser \
     && useradd -m -g sbuser sbuser \
     # Disable ptrace mechanism to avoid heap dumping from other processes
-    && echo "kernel.yama.ptrace_scope = 3" >> /etc/sysctl.d/10-ptrace.conf \
-    # Delete all java.management files to hinder student's ability to retrieve information from the grader
+    && echo "kernel.yama.ptrace_scope = 3" \
+        $( # Delete all java.management files to hinder student's ability to retrieve information from the grader
+        ) >>/etc/sysctl.d/10-ptrace.conf \
```

Hadolint (2.12.0)

[error] unexpected ')'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)